installing/installing_disconnected/installing-disconnected-preparations.adoc

```diff
@@ -1,18 +1,36 @@
 [id="installing-disconnected-preparations"]
-= Preparing to install a disconnected cluster
+= Creating a mirror registry for a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: installing-disconnected-preparations
 
 toc::[]
 
 Before you install a cluster on infrastructure that you provision in a
-disconnected environment, you must prepare the environment.
+disconnected environment, you must create a mirror registry.
 
-//include::modules/cluster-entitlements.adoc[leveloffset=+1]
+.Prerequisites
 
+* You must have access to a bastion host that can access both the internet and
+your network.
```

Comment:
You don't need this to be a bastion host.  You can set up the registry outside, shut it down, move it into your disconnected network, and start it back up.  Or bring images in via other pathways.  Having a registry with access to both sides is just the easiest approach to setting up a mirror.